tools/docker-compose/Dockerfile

```diff
@@ -90,9 +90,12 @@ ADD requirements/requirements.txt \
     requirements/requirements_dev.txt \
     requirements/requirements_ansible_uninstall.txt \
     requirements/requirements_tower_uninstall.txt \
+    requirements/collections_requirements.yml \
     /tmp/requirements/
 RUN mkdir -p /venv && chmod g+w /venv
 RUN cd /tmp && VENV_BASE="/venv" make requirements_dev
+RUN mkdir -p /vendor/inventory_collections && chmod g+w /vendor/inventory_collections
+RUN cd /tmp && COLLECTION_BASE="/vendor/inventory_collections" make requirements_collections
```

Comment:
I kind of feel like it shouldn't be doing this. This is going to trigger lots of runs of this command, and will result in a lot of load on the server.

Reply:
Another side-car container that is just the bits?

Reply:
> Or, we map in that dir and persist the collections on our host machine.

This, I have a solution for this right now. I feel safe in saying that I could not have even have completed this if I didn't have a fast way of checking out branches in checkouts of the respective collections.

I have another PR which was just a set of targets to do a `git clone` of all the needed collections. You still have 1 more step that you have to change the setting for the collection folder (probably useful in other circumstances too). I abused the existing dir mapping by putting it in `awx/plugins/collections`

I want to take that content and put it into the `tools/` folder.

But for this Dockerfile, I guess I take back what I say. There's no way to avoid putting these in the prod openshift image, so we should have it here too for parity between the two.